tor_core/helpers.py

```diff
@@ -94,10 +94,15 @@ def send_to_modchat(message, config, channel='general'):
     :return: None.
     """
     if config.modchat:
-        config.modchat.chat_post_message(
-            message,
-            channel=channel
-        )
+        try:
+            config.modchat.chat_post_message(
+                message,
+                channel=channel
+            )
+        except requests.exceptions.ConnectionError as e:
```

Comment:
Would it make more sense to do a full `except Exception as e:` and then print the exception name here, since this is truly supposed to be a 'fire and forget' method?

Reply:
Good idea. I doubt anything other than a network problem could go wrong here, but we don't want it to kill the bot anyway.